ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/StandaloneExecutorTest.java

```diff
@@ -88,7 +83,13 @@ public void before() throws IOException {
     persistentQueryMetadata = EasyMock.niceMock(PersistentQueryMetadata.class);
     queriesFile = TestUtils.tempFile().getPath();
     standaloneExecutor =
-        new StandaloneExecutor(ksqlConfig, engine, queriesFile, udfLoader, false);
+        new StandaloneExecutor(
+            ksqlConfig,
+            engine,
+            queriesFile,
+            udfLoader,
+            false,
+            niceMock(VersionCheckerAgent.class));
```

Comment:
- pull the mock into a field
- add a test that ensures the executor starts the agent.